addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -78,3 +83,218 @@ def _get_default_payment_method_id(self):
         if self.provider != 'stripe':
             return super()._get_default_payment_method_id()
         return self.env.ref('payment_stripe.payment_method_stripe').id
+
+    # -------------------------------------------
+    # Stripe Connect
+    # -------------------------------------------
+    def _stripe_publishable_key(self):
+        """Gets Stripe Publishable Key
+
+        :returns: Stripe Publishable Key
+        :rtype: str
+        """
+        return self.stripe_publishable_key
+
+    def _stripe_secret_key(self):
+        """Gets Stripe Secret Key
+
+        :returns: Stripe Secret Key
+        :rtype: str
+        """
+        return self.stripe_secret_key
+
+    def _stripe_webhook_secret(self):
+        """Gets Stripe Webhook Secret
+
+        :returns: Stripe Webhook Secret
+        :rtype: str
+        """
+        return self.stripe_webhook_secret
+
+    def _stripe_additional_header(self, endpoint):
+        """Returns an additional header to Stripe API calls
+
+        This method adds the Stripe Account on behalf of which the Stripe call is made.
+
+        :param endpoint: the Stripe endpoint which will be called
+        :returns: An additionnal header dict
+        :rtype: dict
+        """
+        return {}
+
+    def _stripe_webhook_url(self):
+        return self.company_id.get_base_url() + '/payment/stripe/webhook'
+
+    def _stripe_create_checkout_session_values(self):
+        """Returns a dict of additional information to add in the Stripe Session
+
+        :returns: A dict of additional session information
+        :rtype: dict
+        """
+        return {}
+
+    # -------------------------------------------
+    # Stripe Create Webhook
+    # -------------------------------------------
+    def action_stripe_create_webhook(self):
+        self.ensure_one()
+        if self.stripe_secret_key and not self.stripe_webhook_secret:
+            return self._create_webhook() or True
+        return True
+
+    def _create_webhook(self):
```

Comment:
```suggestion
    def _stripe_create_webhook(self):
```
This is important for payment modules because they all share the same models.

Reply:
In fact, couldn't the two methods be merged into the action? So that it returns the action directly.

Reply:
Agree !

Reply:
In the first version, this code was called from two different places. But this architecture is now deprecated.